optimizely/event/event_processor.py

```diff
@@ -226,14 +235,17 @@ def _flush_queue(self):
 
   def process(self, user_event):
     """ Method to process the user_event by putting it in event_queue.
+
     Args:
       user_event: UserEvent Instance.
     """
     if not isinstance(user_event, UserEvent):
       self.logger.error('Provided event is in an invalid format.')
       return
 
-    self.logger.debug('Received user_event: ' + str(user_event))
+    self.logger.debug('Received event of type {} for user {}.'.format(
+      type(user_event).__name__, user_event.user_id)
+    )
```

Comment:
Nit: Do we log this in other SDKs? This could be extremely chatty.

Reply:
Yeah, at least in the Java SDK. It could be chatty, but I feel it is a useful message.